ajax/libs/webtorrent-component/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "webtorrent-component",
+  "filename": "webtorrent-component.min.js",
+  "description": "WebTorrent HTML element.",
+  "author": "Mikeal Rogers",
```

Comment:
Let's add `<mikeal.rogers@gmail.com>` into author field, what do you think?
Because we can provide more information to people, and the information source is https://github.com/mikeal/webtorrent-element/blob/master/package.json#L27.

Thanks.